src/pages/sales-zones-levels/index.js

```diff
@@ -0,0 +1,149 @@
+import React, { useEffect } from 'react'
+import WindowToolbar from 'src/components/Shared/WindowToolbar'
+import * as yup from 'yup'
+import { useContext } from 'react'
+import { RequestsContext } from 'src/providers/RequestsContext'
+import { DataGrid } from 'src/components/Shared/DataGrid'
+import { ResourceIds } from 'src/resources/ResourceIds'
+import toast from 'react-hot-toast'
+import { useResourceQuery } from 'src/hooks/resource'
+import { VertLayout } from 'src/components/Shared/Layouts/VertLayout'
+import { Grow } from 'src/components/Shared/Layouts/Grow'
+import { Fixed } from 'src/components/Shared/Layouts/Fixed'
+import { ControlContext } from 'src/providers/ControlContext'
+import { useForm } from 'src/hooks/form'
+import { SaleRepository } from 'src/repositories/SaleRepository'
+import { useError } from 'src/error'
+
+const SalesZonesLevels = () => {
+  const { getRequest, postRequest } = useContext(RequestsContext)
+  const { platformLabels } = useContext(ControlContext)
+  const { stack: stackError } = useError()
+
+  const { labels: labels, access } = useResourceQuery({
+    datasetId: ResourceIds.SalesZoneLevels
+  })
+
+  useEffect(() => {
+    getGridData()
+  }, [])
+
+  function getGridData() {
+    formik.setValues({
```

Comment:
what the purpose of resetting initial values in getGridData?? initialValues in formik not enough?